src/domain/entities/generic/HashMap.ts

```diff
@@ -82,6 +82,13 @@ export class HashMap<K, V> {
         return this.pickBy(([key, value]) => !pred([key, value]));
     }
 
+    compact(): HashMap<K, NonNullable<V>> {
+        return this.pickBy(([_, value]) => value !== null && value !== undefined) as HashMap<
```

Comment:
Typically we'd write `_key`. 

.compact is a useful generic abstraction, can you create a PR in https://github.com/EyeSeeTea/dhis2-app-skeleton ?

Reply:
Got it.
Thaanks! Sure, but first, should I make compact like the "remove falsy values" or "remove null and undefined values", because I remembered _.compact removes falsy values in lodash right? @tokland